scripts/join_metadata.R

```diff
@@ -9,7 +9,7 @@ parser <- ArgumentParser()
 parser$add_argument('--sample_meta', default= 'sample_meta.csv', help= 'Sample meta data for the sequencing run.')
 parser$add_argument('--lfc', default= 'l2fc.csv', help= 'L2FC data.') # level 4
 parser$add_argument('--collapsed_lfc', default= 'collapsed_l2fc.csv', help= 'Collapsed l2fc data.') # level 5
-parser$add_argument('--sig_cols', default= 'cell_set,treatment,dose,dose_unit,day', 
+parser$add_argument('--sig_cols', default= 'cell_set,pert_name,dose,dose_unit,day',
```

Comment:
I know it doesnt matter much, but should dose and dose_unit have the pert prefix for clarity?